testutil/integration/evmos/network/network.go

```diff
@@ -108,9 +114,14 @@ func (n *IntegrationNetwork) configureAndInitChain() error {
 	// Create funded accounts based on the config and
 	// create genesis accounts
 	coin := sdktypes.NewCoin(n.cfg.denom, PrefundedAccountInitialBalance)
-	genAccounts := createGenesisAccounts(n.cfg.preFundedAccounts)
+	genAccounts := createGenesisAccounts(append(n.cfg.preFundedAccounts, n.funder.Address))
 	fundedAccountBalances := createBalances(n.cfg.preFundedAccounts, coin)
 
+	// append the funders initial balance
+	funderBalances := createFunderBalances(n.funder.Address, append(n.cfg.otherCoinDenom, n.cfg.denom))
```

Comment:
should we remove this and have the FundAccount function received a "sender" param?